pom.xml

```diff
@@ -80,8 +80,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <doxia-markdown.version>1.9</doxia-markdown.version>
-        <com.buschmais.xo.version>1.4.2</com.buschmais.xo.version>
-        <org.neo4j_3x.version>3.5.35</org.neo4j_3x.version>
+        <com.buschmais.xo.version>2.0.0-SNAPSHOT</com.buschmais.xo.version>
```

Comment:
Can we build a release of XO first?

Reply:
Deferred 2.0.0 release of XO until no further changes are required for Neo4j v4 migration

Reply:
I agree with @StephanPirnbaum, but for me it isn't a blocker.

Reply:
Upgraded to XO 2.0.0